src/components/Tokens/TokenDetails/index.tsx

```diff
@@ -255,9 +257,9 @@ export default function TokenDetails({
             />
           </div>
           {tokenWarning && <TokenSafetyMessage tokenAddress={address} warning={tokenWarning} />}
-          {detailedToken && <BalanceSummary token={detailedToken} />}
+          {!isInfoTDPEnabled && detailedToken && <BalanceSummary token={detailedToken} />}
```

Comment:
nit: define a const `showBalanceSummary` and have it equal `!isInfoTDPEnabled && detailedToken`, use that in both locations